portfolio/src/main/webapp/script.js

```diff
@@ -39,7 +40,11 @@ function addRandomGreeting() {
 }
 
 function getServerName() {
-    fetch('/data').then(response => response.json()).then((stats) => {
+    fetch('/data',{method:"POST", headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json'},
```

Comment:
It seems like content type is `'text/plain'` here.

Note: If you start to send more than one field in a form, it may start to make sense to use a JSON payload instead. I.e.` {firstName: 'Tony', lastName: 'Mai', comment: 'blah blah blah'}`